docs/AssembleBogie.md

```diff
@@ -4,12 +4,22 @@ Bogie portion of the suspension links the middle wheel to the rear wheel.
 
 **Estimated Time:** 30 minutes per side = 1 hour total
 
+You will use the 117mm extrusion and the 122mm extrusion for this.
+
+You will also need the Rear Corner, Fixed Knuckle, and Bogie Wheel printed parts.
+
 ![Bogie parts](images/Bogie01-Parts.jpg)
 
+The 117mm extrusion goes to between the Rear Corner and the angled side of the Bogie Wheel part.
```

Comment:
Information is correct, but "goes to between" reads slightly odd to me. Suggest: "connects". (Same suggestion applies to remainder of this file.)